packages/kit/types/index.d.ts

```diff
@@ -865,7 +865,7 @@ export interface RequestEvent<
 	/**
 	 * Additional data made available through the adapter.
 	 */
-	platform: Readonly<App.Platform>;
+	platform: Readonly<App.Platform> | undefined;
```

Comment:
I think you're 'supposed' to use `void` rather than `undefined` (i.e. the type rather than the value):

```suggestion
	platform: Readonly<App.Platform> | void;
```